services/QuillLMS/app/lib/rule_feedback_history.rb

```diff
@@ -25,12 +25,15 @@ def self.exec_query(conjunction:, activity_id:, start_date:, end_date:)
         .joins('LEFT JOIN feedback_histories ON feedback_histories.rule_uid = comprehension_rules.uid AND feedback_histories.prompt_id = prompts.id')
         .joins('LEFT JOIN feedback_history_ratings ON feedback_histories.id = feedback_history_ratings.feedback_history_id')
         .joins('LEFT JOIN feedback_history_flags ON feedback_histories.id = feedback_history_flags.feedback_history_id')
+        .joins('LEFT JOIN feedback_sessions ON feedback_histories.feedback_session_uid = feedback_sessions.uid')
+        .joins('LEFT JOIN comprehension_turking_round_activity_sessions ON feedback_sessions.activity_session_uid = comprehension_turking_round_activity_sessions.activity_session_uid')
```

Comment:
This is pretty  minor, and not necessary, but you could actually put these joins inside an `if` block with the where condition below since these two joins are only needed if you're using that where:
```ruby
if turk_session_id
  query = query.joins('LEFT JOIN feedback_sessions ON feedback_histories.feedback_session_uid = feedback_sessions.uid')
               .joins('LEFT JOIN comprehension_turking_round_activity_sessions ON feedback_sessions.activity_session_uid = comprehension_turking_round_activity_sessions.activity_session_uid')
               .where("comprehension_turking_round_activity_sessions.turking_round_id = ?", turk_session_id)
```